calico_containers/pycalico/datastore.py

```diff
@@ -341,8 +341,11 @@ def add_ip_pool(self, version, pool):
         # If IP in IP is enabled on the pool, ensure that it is enabled
         # globally.
         if pool.ipip:
-            result = self.etcd_client.read(IP_IN_IP_PATH)
-            if result.value != IP_IN_IP_ENABLED:
+            try:
+                result = self.etcd_client.read(IP_IN_IP_PATH)
+            except EtcdException:
```

Comment:
Is this the correct Exception to be catching here?  There are a couple of failure cases here at least.

1) The key does not exist yet in `etcd` (this is the one we care about)
2) We fail to connect to `etcd` at all (we don't want to catch this)